prow/Makefile

```diff
@@ -15,7 +15,7 @@
 all: build test
 
 
-HOOK_VERSION       ?= 0.139
+HOOK_VERSION       ?= 0.140
```

Comment:
@cjwagner bumped hook for prow. Not sure it is the right way though.

I think we can have 2 commits for this PR as each commit indicates different changes.